index.ts

```diff
@@ -113,6 +113,7 @@ export { WindowReference } from './src/app/utilities/window.reference';
 export { NavigationItemConfig } from './src/app/navigation/navigation-item-config';
 export { NavigationModule } from './src/app/navigation/navigation.module';
 export { VerticalNavigationComponent } from './src/app/navigation/vertical-navigation.component';
+export { ApplicationLauncherComponent} from './src/app/navigation/application-launcher.component';
```

Comment:
There is now a new index.ts file in each component directory. This allows a dev to import the module like so:

import { NavigationModule } from 'patternfly-ng/navigation';

Please add this export to src/app/navigation/index.js -- you'll need to pull the latest source.